configure.ac

```diff
@@ -559,6 +559,9 @@ AC_CHECK_HEADERS(fcntl.h limits.h malloc.h sys/file.h sys/ioctl.h sys/time.h sys
 dnl For module/pam_lastlog
 AC_CHECK_HEADERS(lastlog.h utmp.h utmpx.h)
 
+dnl For module/pam_setquota
+AC_CHECK_HEADERS(sys/quota.h linux/quota.h)
+
```

Comment:
You don't need to check these headers if you don't use the result of the check.

Reply:
I see, pam_lastlog has `#ifdef HAVE_UTMP_H` because it can use either utmp.h or lastlog.h.

No such thing is possible for pam_setquota, the queck for quotactl() should be enough then.